tests/testthat/test_eml.R

```diff
@@ -478,6 +478,11 @@ test_that('eml_nsf_to_project fails gracefully', {
 
 })
 
+test+that('eml_nsf_to_project parses two-word last names correctly', {
+  proj <- eml_nsf_to_project("1822406", eml_version = "2.2")
+
+  expect_equal(proj$personnel$individualName$givenName, "name")
```

Comment:
I suspect the Travis checks will fail here. Hint - what are we expecting `proj$personnel$individualName$givenName` to equal to (the second argument)?